package.json

```diff
@@ -19,7 +19,7 @@
   "author": "GoDaddy Operating Company, LLC",
   "license": "MIT",
   "peerDependencies": {
-    "stylelint": ">=14"
+    "stylelint": "^15"
```

Comment:
should this match what is in the devDep? i don't know if this even works on 15 😅